qutebrowser/html/bindings.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
```

Comment:
I think the styling on this page seems a bit off from [some of the other pages qutebrowser has, such as qute:history](https://github.com/mjauvin/qutebrowser/blob/4848182204c234f082e3e508588bc852873c78da/qutebrowser/html/history.html). 

I didn't explore it too much, but doing something like:

```
{% extends "styled.html" %}

{% block style %}
{{super()}}
{% endblock %}
```

in the header is a better start, although I didn't look at it too much to see if anything looked bad.

Reply:
Definitely, I added the core functionality, anyone better with styling is welcome to share!